src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1776,6 +1770,7 @@ class SelectStatementSegment(ansi.SelectStatementSegment):
             Ref("NamedWindowSegment", optional=True),
             Ref("OrderByClauseSegment", optional=True),
             Ref("LimitClauseSegment", optional=True),
+            Ref("OffsetClauseSegment", optional=True),
```

Comment:
According to https://www.postgresql.org/docs/current/queries-limit.html, after limit.